plugins/smart_proxy_hdm/hdm.rb

```diff
@@ -0,0 +1 @@
+gem 'smart_proxy_hdm', '0.1.0'
```

Comment:
you can also drop this file and use this one directly: https://github.com/betadots/smart_proxy_hdm/blob/main/bundler.d/hdm.rb
(and then  using `bundler.d/hdm.rb` in the `install` file below)

Reply:
I think I did that now